### DISCUSSION TITLE
Error with the Make Command

### DISCUSSION THREAD

**Discussion by erdikucuk** [ChaissonLab/danbing-tk#12] [Q&A]
Hi, I get an error when I tried to use the make command in the cloned repo (Linux machine):

```
src/aQueryFasta_thread.h:4:38: fatal error: cereal/archives/binary.hpp: No such file or directory
compilation terminated.
make: *** [bin/vntr2kmers_thread] Error 1

```

However, cereal/archives/binary.hpp does exist and I can ls to it, so I couldn't figure out how to solve this problem.
Thanks a lot in advance!

Best. 
Erdi

**Comment by joyeuxnoel8** [comment]
Hi Erdi,

Thanks for catching the bug. I've pushed a new version, and it should resolve the issue.

-Tony

**Reply by erdikucuk** [reply]
Hi Tony,

Thanks for the prompt response! I saw that you've made the commit in the master branch, but when I re-cloned the repo I still get the same error:

`g++ -std=c++11 -lpthread -O3 -I. -o bin/danbing-tk src/aQueryFasta_thread.cpp
g++ -std=c++11 -O3 -o bin/vntr2kmers_thread src/VNTR2kmers_thread.cpp
In file included from src/VNTR2kmers_thread.cpp:8:0:
src/aQueryFasta_thread.h:4:38: fatal error: cereal/archives/binary.hpp: No such file or directory
compilation terminated.
make: *** [bin/vntr2kmers_thread] Error 1
`
Could it be something on my side? I am not so sure. Again, thanks a lot for helping out!

Best,
Erdi

**Reply by joyeuxnoel8** [reply]
Hi Erdi,

Sorry my bad. Forgot to add Makefile to the commit. Let me know if it still doesn't work.

Best,
-Tony

**Reply by erdikucuk** [reply]
Hi Tony,

Yes, the makefile actually worked for me! Thanks a lot for fixing this.

Best,
Erdi
